content/enterprise/install/automated/automating-the-installer.mdx

```diff
@@ -295,7 +295,7 @@ bash ./install.sh \
     public-address=5.6.7.8
 ```
 
-Note the `private-address` and `public-address` flags provided to the installer.  These may be left out, but the installer will prompt for them if it is unable to determine appropriate values automatically. If the instance will not have a separate public address, you may provide the private address for both values.
+Note the `private-address` and `public-address` flags provided to the installer.  If these are left out the installer will attempt to determine them itself and will prompt for them if it is unable to. If using cloud-init or another non-interactive method to run the installer you will see this prompt in the log output, but the installer will continue to run. If the installer is unable to determine the IP addresses and you do not provide them via the flags, it will appear that the install has been successful, but functionality of Terraform Enterprise will be degraded. For example you may see a `Internal error: SIW-001` error when importing modules to the private registry or when you run the `tfe-admin health-check` command you may see that the Archivist and Vault containers are unhealthy. If the virtual machine instance will not have its own separate public IP address (i.e. you are using a load balancer and a private subnet), you should provide the private IP address for both flags.
```

Comment:
I wonder if there's a more clear way to write this. Do we recommend that people include these? If so I wonder if we should say that directly. Like "We recommend including the flags, x, y, z. If you leave them out...."  Thoughts?

Reply:
oh! we have a recommendation t the end! I wonder if we could start out with that and then explain why.

Reply:
Hi @laurapacilio. The recommendation was specifically about the values in the flags as opposed to always supplying them. I have tried to make it clearer by adding the recommendation at the start and then a new paragraph for the explanation.